#!/usr/bin/env python
"""Visualization config for static layers in the scene graph."""
PACKAGE = "hydra_utils"

import dynamic_reconfigure.parameter_generator_catkin as dr_gen  # NOQA

gen = dr_gen.ParameterGenerator()
gen.add(
    "interlayer_edge_scale",
    dr_gen.double_t,
    0,
    "interlayer edge size",
    0.03,
    0.001,
    1.0,
)
gen.add(
    "interlayer_edge_alpha", dr_gen.double_t, 0, "interlayer edge alpha", 1.0, 0.0, 1.0
)
gen.add(
    "interlayer_edge_use_color", dr_gen.bool_t, 0, "show interlayer edge in color", True
)
gen.add(
    "interlayer_edge_insertion_skip",
    dr_gen.int_t,
    0,
    "Number of edges to skip when drawing interlayer edges",
    0,
    0,
    1000,
)

exit(gen.generate(PACKAGE, PACKAGE, "LayerVisualizer"))
